pkgs/development/libraries/fflas-ffpack/default.nix

```diff
@@ -6,20 +6,29 @@ stdenv.mkDerivation rec {
   name = "${pname}-${version}";
   pname = "fflas-ffpack";
   version = "2.3.2";
+
   src = fetchFromGitHub {
     owner = "linbox-team";
     repo = "${pname}";
     rev = "v${version}";
     sha256 = "1cqhassj2dny3gx0iywvmnpq8ca0d6m82xl5rz4mb8gaxr2kwddl";
   };
+
   checkInputs = [
     gmpxx
   ];
+
   nativeBuildInputs = [
     autoreconfHook
     pkgconfig
-  ] ++ stdenv.lib.optionals doCheck checkInputs;
+  ] ++ stdenv.lib.optionals doInstallCheck checkInputs;
+
```

Comment:
Not sure what github is telling me here. Are you adding whitespace at the start of the line / changing spaces to tabs or something?